src/components/login.jsx

```diff
@@ -1,15 +1,32 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import axios from 'axios';
 
 class Login extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       username: '',
-      password: ''
+      password: '',
+      loginFailed: 'hidden',
     };
   }
 
+  checkLogin(loginObject) {
```

Comment:
I like that this is separating out the login concerns. I also started investigating Passport.js a little bit, so I'm excited to make that happen as well.